patternlab/styleguide/source/assets/scss/03-organisms/_mapped-locations.scss

```diff
@@ -1,59 +1,53 @@
 .ma__mapped-locations {
-  
-  .main-content--full .page-content > & {

```

Comment:
this rule is usually only applied to the patternlab page, to keep the component from running up against the edges of the page. Could you double check it for me please?